collector/bonding_linux.go

```diff
@@ -31,6 +32,7 @@ type bondingCollector struct {
 
 func init() {
 	Factories["bonding"] = NewBondingCollector
+	CollectorsEnabledState["bonding"] = flag.Bool("collectors.bonding.enabled", false, "enables bonding-collector")
```

Comment:
yes, but as far as I see, we don't have them yet for the `main`-package, therefore I'd tried to avoid introducing them and additionally, this would be additional bookkeeping, because we have to keep track of those flags in addition to the collectors, when adding or removing collectors.
This way, the collectors and their "state" is confined to their files, that's the intention behind that and with the map we can easily iterate over the available collectors.
That was the intention behind that.